consensus/pacemaker/pacemaker.go

```diff
@@ -254,11 +261,23 @@ func (p *paceMaker) startNextView(qc *typesCons.QuorumCertificate, forceNextView
 	}
 
 	p.RestartTimer()
-	p.consensusMod.broadcastToNodes(hotstuffMessage)
+
+	anyProto, err := anypb.New(hotstuffMessage)
+	if err != nil {
+		log.Println("[WARN] NewHeight: Failed to convert paceMaker message to proto: ", err)
+		return
+	}
+	p.GetBus().GetConsensusModule().BroadcastMessageToNodes(anyProto)
 }
 
 // TODO(olshansky): Increase timeout using exponential backoff.
 func (p *paceMaker) getStepTimeout(round uint64) timePkg.Duration {
 	baseTimeout := timePkg.Duration(int64(timePkg.Millisecond) * int64(p.pacemakerCfg.GetTimeoutMsec()))
 	return baseTimeout
 }
+
+// HasPacemakerConfig is used to determine if a ConsensusConfig includes a PacemakerConfig without having to cast to the struct
+// (which would break mocks and/or pollute the codebase with mock types casts and checks)
+type HasPacemakerConfig interface {
```

Comment:
Is there a way to avoid this?

Reply:
I don't see a quick solution, I will be investigating.

Reply:
I left a few other comments but this is the only "blocker" I see in this PR.  Let's tend to the other ones and I'll jump in to see if we can solve this too after

Reply:
This is removed with the rebasing on main which includes new config structure.